FAQ.md

```diff
@@ -17,6 +17,40 @@ If you know that your numbers are always in the form &lt;country calling
 code&gt;&lt;national significant number&gt;, it is safe to put a "+" in front to
 indicate this to the library.
 
+### Why does the library treat some non-digit characters as digits?
+
+When parsing, the library does its best to extract a phone number out of the
+given input string. It looks for the phone number in the provided text; it
+doesn't aim to verify whether the string is *only* a phone number.
+
+If the input looks like a vanity number to the library, `parse()` assumes this
+is intentional and converts alpha characters to digits. Please read the
+documentation for
+[`parse()`](http://github.com/googlei18n/libphonenumber/blob/35210c7288b3953360289b24796330bc848dfe2d/java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java#L3040)
```

Comment:
If we refer to master
(https://github.com/googlei18n/libphonenumber/blob/master/java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java)

then it'll always be up to date; on the other hand the line number will change...

I think probably best to have the "master" not the specific version, and tell people to search for parse?